test/transformer/create-mock-values.test.ts

```diff
@@ -14,16 +14,4 @@ describe('create-mock-values', () => {
         expect(properties.property).toBe('sss');
         properties.method();
     });
-
-    it('should create an array of mocks with all the value defined', () => {
```

Comment:
why we're not testing anymore the createMockList with values?

Reply:
I've realised that the test was a duplicate in create-mock-list-values-test

Reply:
ok!